src/Raven.Studio/wwwroot/App/views/database/cmpXchg/editCmpXchg.html

```diff
@@ -1,67 +1,110 @@
-<form data-bind="submit: saveDocument" class="cmp-xchg" autocomplete="off">
-
-    <div class="value-editor flex-window flex-grow content-margin">
-        <div class="flex-window-head">
-            <div class="row">
-                <div class="col-md-6">
-                    <div class="documentId">
-                        <div data-bind="if: isCreatingNewValue()">
-                        <div class="input-group" data-bind="validationElement: key">
-                            <div class="input-group-addon">Compare Exchange Key</div>
-                            <input class="form-control"
-                                   name="documentId"
-                                   id="documentIdInput"
-                                       data-bind="textInput: key, disable: $root.isBusy">
-                            </div>
-                        </div>
-                        <div data-bind="if: !isCreatingNewValue()">
-                            <span class="obj-name" data-bind="text: key"></span>
-                        </div>
-                    </div>
+<form class="cmp-xchg flex-form" data-bind="submit: saveCompareExchangeItem" autocomplete="off">
+    <div class="flex-window flex-grow content-margin">
+        <div class="margin-top">
+            <div class="pull-left-sm btn-group">
+                <button type="submit" class="btn btn-primary" data-bind="enable: isSaveEnabled, css: { 'btn-spinner': spinners.save }">
+                    <i class="icon-save"></i><span>Save</span>
+                </button>
+                <a class="btn btn-default" data-bind="attr: { href: cmpXchUrl }">
+                    <i class="icon-cancel"></i><span>Cancel</span>
+                </a>
+                <button class="btn btn-default" data-bind="click: cloneCmpXch, visible: !isCreatingNewItem()">
+                    <i class="icon-clone"></i><span>Clone</span>
+                </button>
+                <button class="btn btn-danger" data-bind="click: deleteItem, enable: !isCreatingNewItem() && !spinners.delete(), css: { 'btn-spinner': spinners.delete }" >
+                    <i class="icon-trash"></i><span>Delete</span>
+                </button>
+            </div>
+            <div class="pull-right-sm bg-warning padding padding-xs" data-bind="visible: displayExternalChange">
+                <small>
+                    <i class="icon-warning"></i>
+                    <span>Value has been modified outside of the studio. <a href="#" data-bind="click: refresh">Click here to refresh.</a></span>
```

Comment:
does it even work? I'm supprised we have that here.

Reply:
1. `save`  & `delete` cmpExchg may fail if 'index' is not the same.
In these cases, this msg will show to provide a 'refresh' option
2. Actually - need to fix the `refresh `method - to actually fetch